src/main/java/com/sparta/popupstore/domain/popupstore/dto/response/PopupStoreCreateResponseDto.java

```diff
@@ -14,14 +16,16 @@ public class PopupStoreCreateResponseDto {
     private Address address;
     private LocalDate startDate;
     private LocalDate endDate;
+    private List<PopupStoreOperating> popupStoreOperatingList;
```

Comment:
```suggestion
    private List<PopupStoreOperating> operatingList;
```
popupStore 클래스 내에 있으니 그 부분은 생략해도 될 것 같습니다.